metabase/metabase
# Simple mode for datasets, update dataset headers

Updates question header component for datasets and questions based on datasets:

* when a dataset is open, the header is more similar to "Simple Mode" when viewing a table 
* when a dataset is open, its collection is displayed instead of a database name
* when a question based on a dataset is open, the dataset name and its collection are shown instead of a database and tables

### To Verify

1. Open a dataset (you can create one following the steps from #18702)
2. Make the page header has a collection link instead of a database link
2. Add a filter / summarize something
3. Make sure the URL is changed from `/question/:id-:slug` to `/question#hash` (moved to ad-hoc mode)
4. Click the "Save" button
5. Make sure you're not offered to replace the dataset, only to save a new question
6. Save the question
7. Make sure the shows the dataset name and its collection instead of database and tables

### Demo

https://user-images.githubusercontent.com/17258145/141836214-2ea7f8e8-112a-4d07-9aa7-feb90ec0a80d.mov
